core/src/Repository/Mapper/PaginationOffsetLimitMapper.php

```diff
@@ -9,27 +9,25 @@
  * @template TTo
  * @implements Mapper<PaginationOffsetLimit<TFrom>, PaginationOffsetLimit<TTo>>
  */
-Class PaginationOffsetLimitMapper implements Mapper {
-    /**
-     * @param Mapper<TFrom, TTo> $mapper
-     */
-    public function __construct(
-        protected Mapper $mapper,
-    ) {
-    }
+class PaginationOffsetLimitMapper implements Mapper {
+  /**
+   * @param Mapper<TFrom, TTo> $mapper
+   */
+  public function __construct(protected Mapper $mapper) {
+  }
 
-    /**
-     * @param PaginationOffsetLimit<TFrom> $from
-     * @return PaginationOffsetLimit<TTo>
-     */
-    public function map(mixed $from): PaginationOffsetLimit {
-        return new PaginationOffsetLimit(
-            array_map(function ($value) {
-                return $this->mapper->map($value);
-            }, $from->getValues()),
-            $from->getOffset(),
-            $from->getLimit(),
-            $from->getSize(),
-        );
-    }
+  /**
+   * @param PaginationOffsetLimit<TFrom> $from
+   * @return PaginationOffsetLimit<TTo>
+   */
+  public function map(mixed $from): PaginationOffsetLimit {
+    return new PaginationOffsetLimit(
```

Comment:
@orioljp To be enable to debug it, better move the result of array_map to a new variable previously to the return.